src/Components/ToDoList.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Todo from './Todo';
+
+class ToDoList extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            storedList: [],
+            newEntries: ''
+        };
+    }
+
+    handleEntry = (event) => {
+        this.setState({ newEntries: event.target.value });
+    }
+
+    addEntry = (event) => {
+        event.preventDefault();
+       const EntryList = this.state.storedList;
+       EntryList.push(this.state.newEntries);
+        this.setState({
+            storedList: EntryList,            
+            newEntries: ''
+        })
+    }
+
+    render() {
+        return (
+            <div>
+                {this.state.storedList.map(entry => <Todo thing={entry} />)}
```

Comment:
Make sure you're adding `key` to any child components rendered with `map` or any type of iterative loop.